Orange/tests/test_evaluation_testing.py

```diff
@@ -448,8 +451,8 @@ def test_store_data(self):
     def test_store_models(self):
         nrows, ncols = 50, 10
         data = random_data(nrows, ncols)
-        train = data[:80]
-        test = data[80:]
+        train = data[:40]
+        test = data[40:]
```

Comment:
This test is about `store_models=True`. With test data empty, the models now fail. So test data is made non-empty.